components/datepicker/datepicker-inner.ts

```diff
@@ -91,9 +91,8 @@ export class DatePickerInner implements OnInit {
   private onlyCurrentMonth:boolean;
   @Input()
   private shortcutPropagation:boolean;
-  // todo: change type during implementation
   @Input()
-  private customClass:any;
+  private customClass:Array<{date:Date, mode:string, clazz:string}>;
```

Comment:
When I changed type from any, clazz looked more proper. I didn't touch that part of readme.md. Thanks for noticing the difference. Now I updated the readme.md too.